test/promql/tests/test_controller_reconciliation_errors.yaml

```diff
@@ -0,0 +1,75 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
```

Comment:
i suggest splitting this to multiple test cases. Have one test case for each alerting pod, and then we can have another test case for all the non-alerting pods.